insights/client/constants.py

```diff
@@ -28,7 +28,7 @@ class InsightsConstants(object):
     default_branch_info = {'remote_branch': -1, 'remote_leaf': -1}
     docker_image_name = None
     default_cmd_timeout = 600  # default command execution to ten minutes, prevents long running commands that will hang
-    egg_path = 'https://access.redhat.com/insights/static/core/insights-core.egg'
+    egg_path = 'https://cert-api.access.redhat.com/r/insights/static/core/insights-core.egg'
```

Comment:
Sorry, just noticed this.  Shouldn't the egg remain at the old path with the `/r`?

Reply:
That should read *without* the `/r`.